.github/workflows/build.yml

```diff
@@ -90,6 +90,50 @@ jobs:
       with:
         name: debs
         path: qdigidoc4*.*
+  fedora:
+    name: Build on Fedora to ${{ matrix.container }}
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    strategy:
+      matrix:
+        container: ['fedora:33']
+    env:
+      MAKEFLAGS: -j3
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        submodules: recursive
+    - name: Install Build Deps
+      run: dnf groupinstall -y "Development Tools" "Development Libraries" fedora-packager
+    - name: Install Deps
+      run: |
+        dnf install -y \
+          cmake openssl-devel vim-common qt5-qtsvg-devel
```

Comment:
Could not find a package configuration file provided by "Qt5LinguistTools"  with any of the following names:
    Qt5LinguistToolsConfig.cmake
I think you need install also install qt5-linguist package